docs/code/conds/api/cron_kwargs.py

```diff
@@ -13,4 +13,3 @@ def do_complex():
         - On 1st, 2nd and 3rd day of month
         - From February to August every second month
     """
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).